lib/ansible/modules/network/sda/sda_rest.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: sda_rest
+
+short_description: Direct access to the Cisco Software-Defined Access (SDA) controller
+
+version_added: "2.6"
+
+description:
+    - Enables management of the Cisco Software-Defined Access controller, more commonly called DNA Center
+    - More information about the DNA Center API can be found at U(https://developer.cisco.com/site/dna-center-rest-api/)
+
+options:
+    url_username:
+        description:
+            - The username for use in HTTP basic authentication.
+        required: true
+    url_password:
+        description:
+            - The password for use in HTTP basic authentication.
+        required: true
+    hostname:
+        description:
+            - FQDN or IP address of DNA Central server
+        required: true
+    method:
+        description:
+            - The HTTP method of the request.
+            - Using C(delete) is typically used for deleting objects.
+            - Using C(get) is typically used for querying objects.
+            - Using C(post) is typically used for modifying objects.
+        required: yes
+        default: get
+        choices: [ delete, get, post ]
+        aliases: [ action ]
+    path:
+        description:
+            - Directory path to the endpoint. Do not include FQDN specified in C(hostname)
+        required: true
+    timeout:
+        description:
+            - HTTP timeout value
+        required: false
+        default: 30
+    use_proxy:
+        description:
+            - If C(no), it will not use a proxy, even if one is defined in an environment variable on the target hosts.
+        required: false
+        default: false
+    use_ssl:
+        description:
+            - If C(no), it will use HTTP. Otherwise it will use HTTPS.
+        required: false
+        default: true

```

Comment:
For the documentation, booleans should be defined like:
```yaml
  type: bool
  default: 'yes'
```

Reply:
Makes sense, updated.